tests/shared/src/test/scala/epollcat/TcpSuite.scala

```diff
@@ -209,6 +209,33 @@ class TcpSuite extends EpollcatSuite {
       }
   }
 
+  /* Listening to a wildcard is common practice.
+   * "connect()'ing" to a wildcard address is passing strange.
+   * See epollcat Issue #92 for details.
+   */
+  test("bind to IPv4 wildcard then connect") {
+    IOServerSocketChannel
+      .open
+      .evalTap(_.setOption(StandardSocketOptions.SO_REUSEADDR, java.lang.Boolean.TRUE))
+      .evalTap(_.bind(new InetSocketAddress("0.0.0.0", 0)))
+      // If IPv6 active, IPv4 "0.0.0.0" will have been converted to IPv6 "::0"
+      // because the socket fd was pre-allocated as IPv6.
+      .use { server =>
+        IOSocketChannel.open.use { clientCh =>
+          if (true) { // IPv6 connect
+            server.localAddress.flatMap(clientCh.connect(_))
+          } else { // IPv4 connect
```

Comment:
Why the two cases with the hard-coded `true`?

Reply:
To document and allow toggling IPv6/IPv4. 

I can convert the IPv4 case to a /* */ block.  When I had it that way, it took
some verbiage and was even more confusing.   Please advise.  No problems

Thank you for review.  People are going to try tracing this stuff.
changing to comment.

Reply:
I converted the IPv4 option to a block comment. I hope that makes it less confusing.

Whilst chasing this issue, I found it more than handy to have that code readily
available and not to have to re-create it.

Reply:
Makes sense. Thanks for clarifying!